Add files via upload

orlando156 opened the pull request.

orlando156 committed:
Add files via upload

rovalcor commented on ejercicios de mukteck.sql:
```diff
@@ -0,0 +1,31 @@
+
+
+create table transactions(
+id_cliente int not null PRIMARY KEY,
```
El Primary Key debería ser el ID de la transacción, no del cliente. Es decir, debería ir en la columna de `id_transaccion`. Yo recomendaría también que la columna de `id_transaccion` se llamara solo `id`.

rovalcor commented on orlandolazcano_datosago18_st2.xml:
```diff
@@ -0,0 +1 @@
+<mxfile userAgent="Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.84 Safari/537.36" version="9.0.2" editor="www.draw.io" type="google"><diagram id="8317b9ef-bcea-debe-76ef-c1c3cbf0097a" name="Page-1">7Vxbc9soFP41eUzHutp+jL3J7kM6s5N0ZtunDpawxRYLL8K101+/IIEuBl/SyMhx1Ham0jFCcL7v3ATSjTddbv+kYJV8JjHEN+4g3t54f9y4rhM6Hv9PSF4KiT8YFYIFRbFsVAme0S8ohQMpXaMYZo2GjBDM0Kop ...
```
En tu diagrama relacional conectas `transactions` en `id_cliente` con `products_transactions` en `id_transaccion`. Debe conectarse siempre a la columna de referencia correcta. De hecho en tu código se ve reflejado este pequeño detalle.

rovalcor commented on ejercicios de mukteck.sql:
```diff
@@ -0,0 +1,31 @@
+create table products_transactions(
+id_products int not null,
+id_transaccion int NOT null PRIMARY KEY,
+FOREIGN KEY (id_transaccion) references transactions (id_cliente),
```
Lo que estás diciendo aquí es que `id_cliente` está definiendo a la transacción como Primary Key. Eso no suena correcto, y es el error que se ve en tu diagrama. Con que solo corrijas esto, ya estás.

orlando156 committed:
actualizacion de las llaves foranes

modificación en la tabla transactions, haciéndola llave primaria, 

cambiando la relación de la llave foranea en la tabla de products_transactions de id_cliente a id_transaccion

orlando156 committed:
modificación al diagrama

se cambio la conexcio de la tabla transactions y la de la tabla transation _ productos

rovalcor reviewed (approved):
Bien! Solo recuerda las recomendaciones de los que hemos repasado:
* Usar mayúsculas solo en las palabras reservadas
* Usar exclusivamente inglés
* Que los primary keys solo se llamen `id`
